src/nodes/split.h

```diff
@@ -0,0 +1,181 @@
+/* This file is part of onnx2c.
+ *
+ * Split.
+ * "Split a tensor into a list of tensors, along the specified ‘axis’."
+ */
+
+#include <numeric>
+
+namespace toC {
+
+class Split : public Node {
+	public:
+	Split() {
+		op_name = "Split";
+	}
+
+	int64_t axis = 0;
+
+	virtual void parseAttributes( onnx::NodeProto &node ) override
+	{
+		for( const auto& a : node.attribute() ) {
+			if( a.name() == "axis" )
+				axis = parse_attribute_int(a);
+			else
```

Comment:
Here I'd like the code to check for the other attributes too `num_outputs` and `split` (which was deprecated in ONNX Opset 13). It's ok to just error out with an "uninmplemented" if these are given - an explicit error is nicer than a silent failure.